04-data-structures-part1.Rmd

```diff
@@ -477,8 +489,25 @@ fact many R functions which return complex output store their results in a list.
 > ```
 >
 
+> ## Solution to challenge 5: Factors {.challenge}
+> extracting character vector from factor vactor works as we would expect:
```

Comment:
"extracting character vector" -> "extracting a character vector"
and
"vector from factor vactor" -> "vector from a factor vector"